content/en/docs/reference/glossary/deviceclass.md

```diff
@@ -0,0 +1,23 @@
+---
+title: DeviceClass
+id: deviceclass
+date: 2025-05-26
+short_description: >
+  Describes a category of devices in the cluster. Users can claim specific
+  devices in a DeviceClass.
+
+tags:
+- extension
+- core
+---
+ A DeviceClass describes a category of
+{{< glossary_tooltip text="devices" term_id="device" >}} in the cluster that can
+be used with
+[dynamic resource allocation (DRA)](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/).
+
+<!-- more -->
```

Comment:
Is this rendering the glossary page in the preview the way you expect? It looks like most of this content is duplicated in what's behind the `[+]` in the rendered view.

Reply:
Fixed!